agent/engine/engine_unix_integ_test.go

```diff
@@ -70,6 +70,12 @@ const (
 	testIPCNamespaceCommand          = "if [ `ipcs -s | awk '$1+0 == 5' | wc -l` -gt 0 ]; then exit 1; else exit 2; fi;"
 	testIPCNamespaceResourceFound    = 1
 	testIPCNamespaceResourceNotFound = 2
+
+	testGPUImage	      = "nvidia/cuda:9.0-base"
+	testGPUContainerName  = "testGPUContainer"
+	testGPUP2InstanceType = "p2"
```

Comment:
Can you instead make this an array of strings so it's easier to add more instances in the future?

Reply:
Fixed it.